### ISSUE TITLE
Group Tree View Block Defaults to Add a Group When set to Show Active Groups Only

### ISSUE THREAD

**Issue by frogydiak** [SparkDevNetwork/Rock#5515]
### Please go through all the tasks below

- [X] Check this box only after you have successfully completed both the above tasks

### Please provide a brief description of the problem. Please do not forget to attach the relevant screenshots from your side.

When going to Organization group viewer, it defaults to Add Group form. I further check why it is doing that and it seems to be affected by my Group Tree View settings to show only Active Groups.

[Issue 5333](https://github.com/SparkDevNetwork/Rock/issues/5333) was resolved but it created a new issue.

### Expected Behavior

The Group Tree View block should know which is the first Active Group **when the block settings is set to show Active groups** and select it as the active group when rendering the page. See screen capture below.

### Actual Behavior

The Group Tree View block defaults to Add Group (new) form when the page is rendered. See screen capture below.

### Steps to Reproduce

- Go to Organization and set the Group Tree View to show only Active Groups:

![image](https://github.com/SparkDevNetwork/Rock/assets/1373815/8eeba5d3-4479-4fa0-a954-f2df08572512)

- You'll notice that it will open the Add New Group form:

![image](https://github.com/SparkDevNetwork/Rock/assets/1373815/ae89de71-6007-4264-9708-cabfba83863e)

But set it to show All and it works properly.

### Rock Version

15.1

### Client Culture Setting

en-US

**Comment by PraveenMathew92** [comment]
I dont see this happening in the demo site [link](https://rock.rocksolidchurchdemo.com/intranet/org-chart?GroupId=46) which currently runs 15.1, the same version as the one in which the issue is reported.

![image](https://github.com/SparkDevNetwork/Rock/assets/14316941/1c3ff063-7d49-45bd-a3fa-f71d803131db)

The Block does open in Add Mode when all the Groups are inactive. But, that is the default behavior.

![chrome_o1Z6hnz85m](https://github.com/SparkDevNetwork/Rock/assets/14316941/deab9eb4-54fd-49c9-9406-5eb83241fe16)

We are closing this issue for now but please let us know if you have any questions and we would be happy to help.

We appreciate your efforts in helping us make Rock better and would encourage you to report issues in future if you find any.

**Comment by frogydiak** [comment]
@PraveenMathew92 I don't know how to get into it but try this:

![Rock_OrgChart](https://github.com/SparkDevNetwork/Rock/assets/1373815/69680a3f-3409-4af9-ab9a-7af06a86ff55)
